actors/builtin/miner/miner_actor.go

```diff
@@ -734,20 +674,19 @@ func (a Actor) checkPoStProvingPeriodExpiration(rt Runtime) {
 
 	var st State
 	expired := rt.State().Transaction(&st, func() interface{} {
-		if !st.PoStState.isChallenged() {
-			return false // Already exited challenged state successfully prior to expiry.
-		}
 
-		window := power.SurprisePostChallengeDuration
-		if rt.CurrEpoch() < st.PoStState.SurpriseChallengeEpoch+window {
+		window := power.FallbackPostChallengeDuration
+		if rt.CurrEpoch() < st.PoStState.ProvingPeriodStart+ProvingPeriod+window {
+			// TODO: do we ever expect this to happen? seems like it might be a
```

Comment:
it depends on when and how `_rtCheckFallbackPoStExpiry` is triggered

Reply:
choosing to abort for now, leaving a note to say that it needs testing and further consideration